lib/commands/build.ts

```diff
@@ -36,5 +37,14 @@ export class BuildAndroidCommand extends BuildCommandBase implements  ICommand {
 	}
 
 	public allowedParameters: ICommandParameter[] = [];
+
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			if (this.$options.release && (!this.$options.keyStorePath || !this.$options.keyStorePassword || !this.$options.keyStoreAlias || !this.$options.keyStoreAliasPassword)) {
```

Comment:
can you add a check that args.length is 0.
With the code in master, if you try to execute this command with parameters, you receive: "This command does not accept parameters.". But after adding canExecute method, the command service will not check the number of passed parameters and you'll be able to call ```tns build android goshko peshko```